File: src/data/google-cloud/expired.ts
import { LMRouter, Message } from "@lm15/lm15";

const router = new LMRouter();
const request = {
  model: "vertex:gemini-2.5-flash",
  system:
      "You are the field assistant for a wildlife research station. " +
      "Answer in two sentences.",
  messages: [Message.user(
      "What might be eating the acorns under our oak trees at night?",
  )],
};
try {
  console.log((await router.complete(request)).text);
} catch (error) {
  console.log(String(error));
}

File: src/data/google-cloud/expired.rs
use lm15::{LMRouter, Message, Request};

// Dependencies: lm15, tokio (macros, rt-multi-thread)
#[tokio::main]
async fn main() -> Result<(), Box<dyn std::error::Error>> {
    let router = LMRouter::new();
    let request = Request {
        model: "vertex:gemini-2.5-flash".into(),
        system: Some(concat!(
            "You are the field assistant for a wildlife research ",
            "station. Answer in two sentences.",
        ).into()),
        messages: vec![Message::user(concat!(
            "What might be eating the acorns under our oak trees at ",
            "night?",
        ))?],
        ..Default::default()
    };
    match router.complete(&request).await {
        Ok(response) => {
            println!("{}", response.text().unwrap_or_default())
        }
        Err(error) => println!("{error}"),
    }
    Ok(())
}
